### ISSUE TITLE
Make the post similar to instagram

### ISSUE THREAD

**Issue by Tess-Vanta** [riyajha2305/Instagram-Clone#90]
I would like to make the post section card similar to actual Instagram.

![insta](https://user-images.githubusercontent.com/72697074/157100936-5d3c4d42-ebf4-434e-8577-555d9d318264.png)

**Comment by riyajha2305** [comment]
> I would like to make the post section card similar to actual Instagram.
> 
> ![insta](https://user-images.githubusercontent.com/72697074/157100936-5d3c4d42-ebf4-434e-8577-555d9d318264.png)

Do u want to continue on this??

**Comment by Tess-Vanta** [comment]
Yes if available

**Comment by riyajha2305** [comment]
> Yes if available

Yes sure ! Maybe mistakenly I closed this issue. I apologise for this.

**Comment by Tess-Vanta** [comment]
> > Yes if available
> 
> Yes sure ! Maybe mistakenly I closed this issue. I apologise for this.

I think I closed it by mistake 😅
